From 502a306e72ba8c839c69874e7e3adff670cda49a Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Fri, 24 Jun 2022 09:31:41 +0300
Subject: [PATCH] [SW-91832] gaudi2: erom: increase stack size of recovery image

Currently erom recovery stack size is set to 512 bytes which is not
sufficient as erom recovery run image verification code as part of
image validation and will need more stack. This patch increases stack
size of erom recovery image to 4KB.

Change-Id: Ic2c6cb9f6f3b3276b330710068df7ae03d83269c
---

diff --git a/app/ext_rom/recovery_erom.lds.S b/app/ext_rom/recovery_erom.lds.S
index 11cb7e5..c594559 100644
--- a/app/ext_rom/recovery_erom.lds.S
+++ b/app/ext_rom/recovery_erom.lds.S
@@ -28,7 +28,7 @@
 	.stack : {
 		. = ALIGN(8);
 		__stack = . ;
-		. = . + 512 ;
+		. = . + 4096 ;
 		__stack_top = . ;
 	} > .sram
 
